### PR TITLE
Fix kubelet-config file name

### PR DESCRIPTION
**Pull request by steve-hardman**
Changed the kubelet-config file name from `/var/lib/kubelet/config.conf` to `/var/lib/kubelet/config.yaml`

This will fix https://github.com/kubernetes/website/issues/43213

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md [hunk 1]
```diff
@@ -264,14 +264,14 @@ the set of node specific patches must be updated accordingly.
 
 #### Persisting kubelet reconfiguration
 
-Any changes to the `KubeletConfiguration` stored in `/var/lib/kubelet/config.conf` will be overwritten on
+Any changes to the `KubeletConfiguration` stored in `/var/lib/kubelet/config.yaml` will be overwritten on
 `kubeadm upgrade` by downloading the contents of the cluster wide `kubelet-config` ConfigMap.
-To persist kubelet node specific configuration either the file `/var/lib/kubelet/config.conf`
+To persist kubelet node specific configuration either the file `/var/lib/kubelet/configyaml`
```
Comment (dipesh-rawat) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md:269]: Minor correction needed.
```suggestion
To persist kubelet node specific configuration either the file `/var/lib/kubelet/config.yaml`
```

**Comment by steve-hardman** [comment]
Thanks @dipesh-rawat I have merged the suggestion code. Please feeback and review when you have time.

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!
lgtm

[Preview Page](https://deploy-preview-43231--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure/)

**Comment by tengqm** [comment]
Please try squash the PR into a single commit.
Each time when you make amendments to your existing PR, you can do `git commit --amend` to avoid a new commit.
Having more than one commit may pose some annoying problems when the maintainers try to merge the `main` branch with the `dev-1.xx` branch. Unless there is a good reason to split a PR into several commits. When you have to do that, it may mean you are supposed to propose a separate PR.
/approve

**Comment by steve-hardman** [comment]
Thanks. I have done this one commit now.

**Pull request merged**